apps/leave-app/webapp/src/view/LeadReport/LeadReport.tsx

```diff
@@ -14,8 +14,35 @@
 // specific language governing permissions and limitations
 // under the License.
 
-import Title from "@root/src/component/common/Title";
+import { Stack } from "@mui/material";
+import { Dayjs } from "dayjs";
+import dayjs from "dayjs";
+
+import { useState } from "react";
+
+import { PAGE_MAX_WIDTH } from "@root/src/config/ui";
+import { LeadReportResponse } from "@root/src/types/types";
+
+import LeadReportTable from "./component/LeadReportTable";
```

Comment:
Shall we use the imports from aliases and not relative ones?

Also it shouldn't be `@root`. We can fix this in later pr.